package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "xlayers",
-  "version": "1.0.0-beta.4",
+  "version": "1.0.0",
```

The version should remain `1.0.0-beta.4`
```suggestion
  "version": "1.0.0-beta.4",
```